test/powershell/Modules/Microsoft.WSMan.Management/CredSSP.Tests.ps1

```diff
@@ -0,0 +1,48 @@
+Describe "CredSSP cmdlet tests" -Tags 'Feature','RequireAdminOnWindows' {

```

Comment:
I suppose I can combine them.  I thought to separate them because at least one test case requires non-admin.